### ISSUE TITLE
demo bug

### ISSUE THREAD

**Issue by gultekinmg** [kittykatattack/sound.js#30]
Uncaught ReferenceError: module is not defined
    at sound.js:160

**Comment by shaise** [comment]
yes, this happens to me as well, both in the demo page and my custom tests. any solution?

**Comment by Jibbon** [comment]
Met too. Any word on this?

**Comment by badtod** [comment]
Open sound.js

Find:

`var exports = module.exports = {};`

Replace with:

`var exports = {};`

**Comment by rthawkcom** [comment]
This fixes the problem.  Why hasn't it been added to the repo?
